ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-periodics.yaml

```diff
@@ -1387,18 +1583,7 @@ periodics:
     ci-operator.openshift.io/variant: rosacli
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-sts-upgrade-e2e-f7
-  reporter_config:
-    slack:
-      channel: '#ocm-qe-prow-ci-jobs'
```

Comment:
Remove slack notification here is expected ?

Reply:
Missed this. Let me add it back. Thanks